core/src/test/java/org/infinispan/eviction/impl/ObjectEvictionFunctionalTest.java

```diff
@@ -0,0 +1,17 @@
+package org.infinispan.eviction.impl;
+
+import org.infinispan.configuration.cache.StorageType;
+import org.testng.annotations.Test;
+
+/**
+ * @author vjuranek
+ * @since 9.2
+ */
+@Test(groups = {"functional", "smoke"}, testName = "eviction.ObjectEvictionFunctionalTest")
+public class ObjectEvictionFunctionalTest extends BaseEvictionFunctionalTest {
+
+   @Override
+   protected StorageType getStorageType() {
+      return StorageType.OBJECT;
```

Comment:
We should be able to make this into a `@Factory` method similar to https://github.com/infinispan/infinispan/blob/master/core/src/test/java/org/infinispan/container/offheap/OffHeapSingleNodeExpirationEvictionTest.java#L45.